src/Components/ProgressBar/ProgressBar.tsx

```diff
@@ -14,22 +14,17 @@ const ProgressBar = ({ step }: ProgressBarProps) => {
   const { theme, formData } = useContext(Context);
   const totalSteps = getStepDirectory(formData.immutableReferrer).length + STARTING_QUESTION_NUMBER;
   const { id } = useParams();
-  let stepValue: number = 0;
-  if (step !== undefined) {
-    stepValue = step;
-  } else if (id !== undefined) {
-    stepValue = Number(id);
-  }
+
+  let stepValue = step ?? id ?? 0;
+  let progressPercentage: number = ((Number(stepValue) - 1) / totalSteps) * 100;
 
   const progressBarStyles = {
     marginBottom: '5px',
-    backgroundColor: '#d6d6d6c4',
-    border: '1px solid #B0B0B0',
+    backgroundColor: theme.secondaryBackgroundColor,
     borderRadius: '500rem;',
     height: '1rem',
-    boxShadow: 'inset -1px 1px 3px rgb(0 0 0 / 0.2)',
     '& .MuiLinearProgress-bar': {
-      background: `linear-gradient(90deg, ${theme.primaryColor} 0%, ${theme.secondaryColor} 100%)`,
+      background: theme.terraCottaColor,
```

Comment:
Can we use `theme.secondaryColor`?

Reply:
Color updated. Current state of the progress bar:

<img width="843" alt="Screenshot 2023-12-07 at 4 13 24 PM" src="https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/b7371f3b-80ef-4785-b43e-ef42aec917cd">